.travis.yml

```diff
@@ -108,9 +108,9 @@ script:
     - /tmp/julia/bin/julia --precompiled=no -e 'true' &&
         /tmp/julia/bin/julia-debug --precompiled=no -e 'true'
     - /tmp/julia/bin/julia -e 'versioninfo()'
-    - export JULIA_CPU_CORES=2 && export JULIA_TEST_MAXRSS_MB=600 && cd /tmp/julia/share/julia/test &&
```

Comment:
Is it not better to test `Base.runtests()`, which invokes `runtests.jl`, and which is how a user would run the tests?

That also avoids hard-coding the current schema for VERSDIR in `.travis.yml`.